### ISSUE TITLE
$asyncComputed is undefined in computed property

### ISSUE THREAD

**Issue by johannes-z** [foxbenjaminfox/vue-async-computed#56]
I wanted to use `$asyncComputed.posts.error` in a computed property, but it's undefined and non-reactive. Is there a way to accomplish that?

**Comment by foxbenjaminfox** [comment]
This should be fixed as of #58.

**Comment by johannes-z** [comment]
@foxbenjaminfox 

It's still initially `undefined`, at least when the asyncComputed property has `lazy: true` set. Because of this I can't make computed properties depend on the asyncComputed state, e.g. `updating`.

```
computed: {
  computedProp () {
    console.log(this.$asyncComputed.asyncProp) // prints undefined
    return ''
  }
},
asyncComputed: {
  asyncProp: {
    lazy: true,
    default: () => [],
    async get () {
      return []
    }
  }
}
